src/components/PauseSubscriptionPopup/PauseSubscriptionPopup.js

```diff
@@ -86,8 +88,8 @@ const PauseSubscriptionPopup = ({
   };
 
   const closePopupAndRefresh = () => {
-    hideInnerPopup();
-    updateList();
+    dispatch(hidePopup());
+    dispatch(updateList());
```

Comment:
same as above - one action? ;)

Reply:
hidePopup is action from `popupSlice` and updateList is from the `planDetailsSlice` so dont know if we can/should combine them in one action. What do you think?

Reply:
make sens :D lets keep as it is ;)